apps/schedule/schedule_json.py

```diff
@@ -0,0 +1,128 @@
+
+    return days.values()
+
+
+def export_frab_json(schedule):
+    duration_days = ceil((event_end() - event_start()).total_seconds() / 86400)
+
+    rooms = set([proposal.scheduled_venue.name for proposal in schedule])
+
+    schedule_json = {
+        "version": "1.0-public",
+        "conference": {
+            "acronym": "emf{}".format(event_year()),
+            "days": [],
+            "daysCount": duration_days,
+            "end": event_end().strftime("%Y-%m-%d"),
+            "rooms": [
+                {
+                    "name": room,
+                }
+                for room in sorted(rooms)
+            ],
+            "start": event_start().strftime("%Y-%m-%d"),
+            "time_zone_name": str(event_tz),
+            "timeslot_duration": "00:10",
+            "title": "Electromagnetic Field {}".format(event_year()),
+            "url": external_url(".main"),
+        },
+    }
+
+    for day in events_per_day_and_room(schedule):
+        day_schedule = {
+            "date": day["start"].strftime("%Y-%m-%d"),
+            "day_end": day["start"].isoformat(),
+            "day_start": day["end"].isoformat(),
+            "index": day["index"],
+            "rooms": {},
+        }
+        for room, events in sorted(day["rooms"].items()):
+            day_schedule["rooms"][room] = []
+            for proposal in events:
+                links = {
+                    proposal.c3voc_url,
+                    proposal.youtube_url,
+                    proposal.thumbnail_url,
+                    proposal.map_link,
+                }
+                links.discard(None)
+                links.discard("")
+                day_schedule["rooms"][room].append(
+                    {
+                        "abstract": None,  # The proposal model does not implement abstracts
+                        "attachments": [],
+                        "date": event_tz.localize(proposal.start_date).isoformat(),
+                        "description": proposal.description,
+                        "do_not_record": proposal.video_privacy != "public",
+                        "duration": get_duration(proposal.start_date, proposal.end_date),
+                        "guid": None,
+                        "id": proposal.id,
+                        # This assumes there will never be a non-english talk,
+                        # which is probably fine for a conference in the UK.
+                        "language": "en",
+                        "links": sorted(links),
+                        "persons": [
+                            {
+                                "name": name.strip(),
+                                "public_name": name.strip(),
+                            }
+                            for name in (proposal.published_names or proposal.user.name).split(",")
+                        ],
+                        "recording_license": "CC BY-SA 3.0",
```

Comment:
We're at 4.0 now - see #1643 

Would probably be good to set this in a single place. I expect a constant or config item would be fine!

Reply:
Would propose i'll create a separate PR for that